lib/rucio/daemons/bb8/common.py

```diff
@@ -101,12 +100,79 @@ def rebalance_rule(parent_rule_id, activity, rse_expression, priority, source_re
                           comment=parent_rule['comments'] if not comment else comment,
                           ask_approval=False,
                           asynchronous=False,
+                          ignore_account_limit=True,
                           priority=priority)[0]
 
-    update_rule(rule_id=parent_rule_id, options={'child_rule_id': child_rule, 'lifetime': 0})
+    update_rule(rule_id=parent_rule['id'], options={'child_rule_id': child_rule, 'lifetime': 0})
     return child_rule
 
 
+def _list_rebalance_rule_candidates_dump(rse, mode=None):
+    """
+    Download dump to tmporary directory
+
+    :param rse:          RSE of the source.
+    :param mode:         Rebalancing mode.
+    """
+
+    from requests import get
+
+    # get last sunday locks dump date:
+    today = date.today()
+    today_idx = (today.weekday() + 1) % 7
+    sun = today - timedelta(today_idx)
+    last_sunday_date = sun.strftime('%d-%m-%Y')
+
+    # expected location of a dump for given rse
+    dumps_location = config_get('bb8', 'dumps_location', raise_exception=False, default='http://rucio-analytix.cern.ch:8080/LOCKS/')
+    rse_dump = '%sGetFileFromHDFS?date=%s&rse=%s' % (dumps_location, str(last_sunday_date), rse)
```

Comment:
I am not sure if we should do it like that.
This way of getting the dump is quite specific to how ATLAS stores the logfiles on rucio-analytix. I think it might be better if BB8 just takes a dump (or URL to a dump) and all this dump identification part we do outside. This way it is easier for non-ATLAS users to use bb8.

Reply:
uff ... I wouldn't like to set some probe outside of bb8, on which the bb8 would be dependent.